sklearn/neighbors/lof.py

```diff
@@ -118,14 +119,22 @@ class LocalOutlierFactor(NeighborsBase, KNeighborsMixin, UnsupervisedMixin):
     n_neighbors_ : integer
         The actual number of neighbors used for :meth:`kneighbors` queries.
 
+    offset_ : float
+        Offset used to define the decision function from the raw scores.
```

Comment:
We should maybe not mention decision function here as well since it is private. "Offset used to predict outliers from the raw scores"?

Reply:
You are right!